backend/src/main/resources/application.properties

```diff
@@ -13,7 +13,8 @@ spring.flyway.baseline-on-migrate=true
 logging.level.ch.puzzle.okr=INFO
 logging.level.org.springframework.security=INFO
 logging.level.org.flywaydb.core=INFO
-
+logging.level.com.zaxxer.hikari.HikariConfig=INFO
```

Comment:
Nit: The application.properties is the last default we configure in the application. Usually we want to have the `application.properties` to match the config of the `application-prod.properties` because if we miss something there the default acts as a sane fallback.